addons/webinterface.default/js/kodi-webinterface.js

```diff
@@ -8053,7 +8053,7 @@ this.Kodi.module("AddonApp.Pvr", function(Pvr, App, Backbone, Marionette, $, _)
   API = {
     isEnabled: function() {
       return App.request("addon:isEnabled", {
-        type: 'xbmc.pvrclient'
+        type: 'kodi.pvrclient'
```

Comment:
This is not the upstream repo. It has to be changed in the upstream repo.

Reply:
Have you a link to them?

Reply:
https://github.com/xbmc/xbmc/blob/4c8d581a763870539d4de3e69c8f557be01e7086/addons/webinterface.default/addon.xml#L16

Reply:
The master there only apply to master here? And 18.x there only related to Leia?

Another think where I asked in slack about (not complete answered), how is the control on our website about e.g. here https://kodi.tv/addons/browse?keyword=&category=1751&author=&sort=title+ASC has not found on his code something related to this name.